dashboard/app/models/curriculum_reference.rb

```diff
@@ -0,0 +1,42 @@
+# == Schema Information
+#
+# Table name: levels
+#
+#  id                       :integer          not null, primary key
+#  game_id                  :integer
+#  name                     :string(255)      not null
+#  created_at               :datetime
+#  updated_at               :datetime
+#  level_num                :string(255)
+#  ideal_level_source_id    :integer
+#  solution_level_source_id :integer
+#  user_id                  :integer
+#  properties               :text(65535)
+#  type                     :string(255)
+#  md5                      :string(255)
+#  published                :boolean          default(FALSE), not null
+#  notes                    :text(65535)
+#
+# Indexes
+#
+#  index_levels_on_game_id  (game_id)
+#  index_levels_on_name     (name)
+#
+
+class CurriculumReference < External
```

Comment:
Any reason to make this a legacy DSLDefined level instead of using the `.level`format?  We're trying to avoid creating new DSL types and eventually (long time from now) want to migrate everything to use `.level`.

Here's an example where we converted DSLDefined levels to not inherit from DSLDefined: https://github.com/code-dot-org/code-dot-org/pull/3762/commits/2e328306e11c54adf6b4d6b262296a52b00f8489.